aeron-client/src/main/cpp/util/aeron_error_client.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2014-2020 Real Logic Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "util/aeron_error_client.h"
+
+#if defined(AERON_COMPILER_MSVC)
+static pthread_key_t error_key = TLS_OUT_OF_INDEXES;
+
+BOOL aeron_error_dll_process_attach()
```

Comment:
What prevents these functions from name clashing with the existing functions in the C native driver? If the C native driver were brought in, this seems like it would break.

Reply:
This will be addressed